scripts/ci/_wait-for.sh

```diff
@@ -21,21 +21,33 @@ USAGE
   exit "$exitcode"
 }
 
+check_availability() {
+  nc -z "$HOST" "$PORT" > /dev/null 2>&1
+
+  result=$?
+  if [ $result -eq 0 ] ; then
+    if [ $# -gt 0 ] ; then
+      exec "$@"
+    fi
+    exit 0
+  fi
+  sleep 1
```

Comment:
strange place for this sleep tbh. maybe pull it up in `wait_for` where the flow logic is?